tests/acceptance/features/bootstrap/AuthContext.php

```diff
@@ -884,11 +887,12 @@ public function administratorRequestsURLWithUsingBasicAuth(string $url, string $
 	 */
 	public function userRequestsURLWithUsingBasicTokenAuth(string $user, string $url, string $method):void {
```

Comment:
NOTE: This function is used in the Given steps below which should be prevented. But do it in the next iteration. See the second point todo list (https://github.com/owncloud/ocis/issues/7082). First, fix all `setResponse` usage and then check the reuse of step-def functions in other steps and refactor it.